Remove the hardcoded dependencies.
they are platform specific and not needed.

https://github.com/guard/listen/pull/54

--- listen-2.7.11/listen.gemspec~	2014-09-26 20:31:45.000000000 +0300
+++ listen-2.7.11/listen.gemspec	2014-12-14 19:41:38.193631843 +0200
@@ -22,8 +22,6 @@
   s.required_ruby_version = '>= 1.9.3'
 
   s.add_dependency 'celluloid',  '>= 0.15.2'
-  s.add_dependency 'rb-fsevent', '>= 0.9.3'
-  s.add_dependency 'rb-inotify', '>= 0.9'
 
   s.add_development_dependency 'bundler', '>= 1.3.5'
   s.add_development_dependency 'celluloid-io', '>= 0.15.0'
